server/handlers_test.go

```diff
@@ -227,3 +229,75 @@ func TestHandleAuthCode(t *testing.T) {
 		})
 	}
 }
+
+func mockConnectorDataTestStorage(t *testing.T, s storage.Storage) {
+	c := storage.Client{
+		ID:           "test",
+		Secret:       "barfoo",
+		RedirectURIs: []string{"foo://bar.com/", "https://auth.example.com"},
+		Name:         "dex client",
+		LogoURL:      "https://goo.gl/JIyzIC",
+	}
+
+	err := s.CreateClient(c)
+	require.NoError(t, err)
+
+	c1 := storage.Connector{
+		ID:   "test",
+		Type: "mockPassword",
+		Name: "mockPassword",
+		Config: []byte(`{
+"username": "test",
+"password": "test"
+}`),
+	}
+
+	err = s.CreateConnector(c1)
+	require.NoError(t, err)
+}
+
+func TestConnectorDataNotEmpty(t *testing.T) {
```

Comment:
Let's give this test a more specific name like the `TestPasswordConnectorDataNotEmpty`.

Reply:
I renamed the test :)